package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ibm-cloud/secrets-manager",
-  "version": "2.0.5",
+  "version": "2.0.6",
```

Comment:
This change should be removed. Publication automation bumps version automatically.